sroka/api/google_ad_manager/gam_api.py

```diff
@@ -92,3 +92,65 @@ def get_data_from_admanager(query, dimensions, columns, start_date, end_date, cu
     with gzip.open(report_file.name) as file:
         data = pd.read_csv(file)
     return data
+
+
+def get_users_from_admanager(query, dimensions, network_code=None):
+    
+    user_df = pd.DataFrame()
+    
```

Comment:
Please remove whitespaces from blank lines. You can use flake8 (as advised in README.md) to track where they are. While you're on it could you remove those in the whole file? Because I can see that there are more than only in the new part.

Reply:
Hey @martynaut, I do not get any warnings or errors regarding whitespaces while checking code with flake8.
I see errors E501 (line too long), but that's something else. I've checked mentioned blank lines and I do not see issues. Happy to fix it if the problem still exists

Reply:
If you checked and there are no spaces now then we can resolve it. When I checked there were some. Thank you!